### PR TITLE
make: Remove CONTAINER_ENGINE_FULL, use QUIET and CONTAINER_ENGINE

### PR DESCRIPTION
**Pull request by vadorovsky**
CONTAINER_ENGINE_FULL was defined as $(QUIET)$(CONTAINER_ENGINE), but
having that variable was confusing and inconsistent. This change removes
it and uses $(QUIET)$(CONTAINER_ENGINE) (or just CONTAINER_ENGINE it
it's not a beginning of the command).

Signed-off-by: Michal Rostecki <mrostecki@opensuse.org>

### CODE REVIEW

**Comment by mrostecki** [comment]
test-me-please

**Comment by mrostecki** [comment]
test-me-please

File: Makefile [hunk 1]
```diff
@@ -213,7 +213,7 @@ GIT_VERSION: .git
 docker-cilium-image-for-developers:
 	# DOCKER_BUILDKIT allows for faster build as well as the ability to use
 	# a dedicated dockerignore file per Dockerfile.
-	DOCKER_BUILDKIT=1 $(CONTAINER_ENGINE_FULL) build \
+	DOCKER_BUILDKIT=1 $(CONTAINER_ENGINE) build \
```
Comment (joestringer) [inline Makefile]: maybe this one can get `$(QUIET)` (at the beginning of the line) too?

File: Makefile [hunk 2]
```diff
@@ -270,16 +270,16 @@ docker-plugin-manifest:
 	$(QUIET) contrib/scripts/push_manifest.sh docker-plugin $(DOCKER_IMAGE_TAG)
 
 docker-image-runtime:
-	cd contrib/packaging/docker && ${CONTAINER_ENGINE} build --build-arg ARCH=$(GOARCH) -t "cilium/cilium-runtime:$(UTC_DATE)" -f Dockerfile.runtime .
-	${CONTAINER_ENGINE} tag cilium/cilium-runtime:$(UTC_DATE) cilium/cilium-runtime:$(UTC_DATE)-${GOARCH}
+	cd contrib/packaging/docker && $(CONTAINER_ENGINE) build --build-arg ARCH=$(GOARCH) -t "cilium/cilium-runtime:$(UTC_DATE)" -f Dockerfile.runtime .
```
Comment (joestringer) [inline Makefile:274]: Ooh, good spotting.

**Review by joestringer — COMMENTED** [review]
Minor nit, I realise you're still drafting this but I glanced through already :)

File: Makefile (hunk 1 above)
Comment (vadorovsky) [reply Makefile]: I was wondering what I should do with this one. I was not sure how `@` interacts with environment variables. But now I have an answer that mixing those is fine. ;)

**Comment by mrostecki** [comment]
test-me-please

**Comment by mrostecki** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

**Comment by mrostecki** [comment]
test-me-please

**Review by qmonnet — APPROVED** [review]

**Pull request merged**